timemachine/fe/free_energy.py

```diff
@@ -32,11 +33,24 @@ def __init__(self, omm_system, conf, box):
         self.box = box
 
 
-@dataclass
+@dataclass(frozen=True)
 class MDParams:
     n_frames: int
     n_eq_steps: int
     steps_per_frame: int
+    local_steps: int = 0
+    k: float = 1_000.0
+    min_radius: float = 1.0
```

Comment:
\# (nm)

Reply:
Fixed in [6f2ae30](https://github.com/proteneer/timemachine/pull/1026/commits/6f2ae30b84861f20fee58791fa241e945776728c)